packages/location-management/package.json

```diff
@@ -35,15 +35,18 @@
   "license": "Apache-2.0",
   "private": false,
   "dependencies": {
-    "@opensrp/reducer-factory": "^0.0.11"
+    "uuidv4": "^6.2.5",
```

Comment:
@p-netm @kelvin-muchiri @KipSigei 
should this be peer dependency or direct despondency ? cuz we are using this pacakges in several other packages too.

Reply:
@ashfahan The `@opensrp/reducer-factory` package? I think also too

Reply:
ok